doc/releases/release-notes-3.5.rst

```diff
@@ -212,6 +210,12 @@ Trusted Firmware-M
 
 * Timer
 
+  * The TI CC13/26xx system clock timer compatible was changed from
+    :dtcompatible:`ti,cc13xx-cc26xx-rtc` to :dtcompatible:`ti,cc13xx-cc26xx-rtc-timer`
+    and the corresponding Kconfig option from :kconfig:option:`CC13X2_CC26X2_RTC_TIMER`
+    to :kconfig:option:`CC13XX_CC26XX_RTC_TIMER` for improved consistency and
+    extensibility. No action is required unless the internal timer was modified.
```

Comment:
```suggestion
    extensibility. No further action is required unless the internal timer was modified.
```

Reply:
Actually, there is really no action required at all for almost all users so I personally wouldn't use the word "further" here.

Background: The Kconfig option is selected by default. It's quite improbable that someone changed it in prj.conf as this would break the mandatory system clock and the kernel would stop working (unless you implemented your own kernel system clock - but then you know what you're doing anyway).

Maybe you thought - as I did in the beginning - that this has something to do with rtc.h or counter.h? Making such confusion less probable was one of my intentions for the change.

Reply:
@kartben can you please confirm if you are aligned with the above explanation?